presto-docs/src/main/sphinx/release/release-0.286.rst

```diff
@@ -0,0 +1,88 @@
+_________________________------------------xxx---------
+* Upgrade Apache Iceberg to 1.4.3.------------------xxx---------
+
+SPI Changes
+___________
+* Add support for connectors to return joins in ``ConnectorPlanOptimizer.optimize``. :pr:`21605`
+
+Hive Changes
+____________
+* Fix parquet dereference pushdown which was not working unless the ``parquet_use_column_names`` session property was set. :pr:21647
+* Fix CTE materialization for unsupported Hive bucket types. :pr:`21549`
+* Remove hive config ``hive.s3.use-instance-credentials`` as deprecated. :pr:`21648`
+
+Hudi Changes
+____________
+* Upgrade Hudi version to 0.14.0. :pr:`21012`
+
+Iceberg Changes
+_______________
+* Upgrade Apache Iceberg to 1.4.3.
+* Add Iceberg Filter Pushdown Optimizer Rule for execution with Velox. :pr:`20501`
+* Add ``iceberg.pushdown-filter-enabled`` config property to Iceberg Connector. This config property controls the behaviour of Filter Pushdown in the Iceberg connector. :pr:`20501`
+* Add `register <../connector/iceberg.html#register-table>`_  and `unregister <../connector/iceberg.html#unregister-table>`_ procedures for Iceberg tables. :pr:`21335`
+* Add session property ``iceberg.delete_as_join_rewrite_enabled`` (enabled by default) to apply equality deletes as a join. :pr:21605
+* Add support for querying ``"$data_sequence_number"`` which returns the Iceberg data sequence number of the file containing the row. :pr:`21605`
+* Add support for querying ``"$path"`` which returns the file path containing the row. :pr:`21605`
+* Add support for reading v2 row level deletes in Iceberg connector. :pr:`21189`
+* Add support for Day, Month, and Year transform function with partition column for date type in Presto Iceberg connector. :pr:`21303`
+* Add support for Day, Month, and Year transform function with partition column for timestamp type in Presto Iceberg connector. :pr:`21303`
+* Day, Month, Year & Hour partition column transform functions are supported when altering a table to add partition columns in Presto Iceberg connector. :pr:`21575`
+* Optimize Table Metadata calls for Iceberg tables. :pr:`21629`
+* SELECT * FROM tab1 FOR SYSTEM_TIME AS OF TIMESTAMP '2023-10-17 13:29:46.822 America/Los_Angeles';. ----------xxxx-------------
+* SELECT * FROM tab1 FOR SYSTEM_VERSION AS OF 8772871542276440693;.----------xxxx-------------
+* SELECT * FROM tab1 FOR TIMESTAMP AS OF TIMESTAMP '2023-10-17 13:29:46.822 America/Los_Angeles';. ----------xxxx-------------
+* Add support for `time travel <../connector/iceberg.html#time-travel-using-version-system-version-and-timestamp-system-time>`_, enabling the retrieval of historical data with the `AS OF` syntax. :pr:`21425`
+* Add support for `Time travel <../connector/iceberg.html#time-travel-using-version-system-version-and-timestamp-system-time>`_ ``TIMESTAMP (SYSTEM_TIME)`` syntax includes timestamp-with-time-zone data type. It will return data based on snapshot with matching timestamp or before. :pr:`21425`
```

Comment:
```suggestion
* Add support for `time travel <../connector/iceberg.html#time-travel-using-version-system-version-and-timestamp-system-time>`_ ``TIMESTAMP (SYSTEM_TIME)`` syntax includes timestamp-with-time-zone data type. It will return data based on snapshot with matching timestamp or before. :pr:`21425`
```
Capitalization consistent with line 81.